src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -353,7 +324,10 @@ export class NativeStakingMapper {
       safeAddress: args.safeAddress,
       validatorsPublicKeys: this.splitPublicKeys(publicKeys),
     });
-    return stakes.reduce((acc, stake) => acc + Number(stake.rewards), 0);
+    return stakes.reduce((acc, stake) => {
+      const netValue = stake.net_claimable_consensus_rewards ?? '0';
```

Comment:
nit: If `net_claimable_consensus_rewards` is always expected to be a number, could we update the schema to automatically parse it as such?"